lib/jekyll/converters/scss.rb

```diff
@@ -209,6 +251,16 @@ def filename
         File.join(site_source_relative_from_pwd, sass_page.name)
       end
 
+      # The path of the input scss (or sass) file. This information will be used for error
+      # reporting and will written into the source map file as main source.
+      #
+      # Returns the path of the input file or nil if #associate_page failed
+      def file
```

Comment:
Since there's a method `:filename` that utilizes `sass_page.name`, let this be called `:file_path` as it utilizes `sass_page.path`.